keyboards/sixkeyboard/keymaps/via/keymap.c

```diff
@@ -0,0 +1,31 @@
+
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+  [0] = LAYOUT(
+    KC_A, KC_B, KC_C, \
+    KC_D, KC_E, KC_F  \
+  ),
+
+  [1] = LAYOUT(
+    KC_TRNS, KC_TRNS, KC_TRNS, \
+    KC_TRNS, KC_TRNS, KC_TRNS  \
+  ),
+
+  [2] = LAYOUT(
+    KC_TRNS, KC_TRNS, KC_TRNS, \
+    KC_TRNS, KC_TRNS, KC_TRNS  \
+  ),
+
+  [3] = LAYOUT(
+    KC_TRNS, KC_TRNS, KC_TRNS, \
+    KC_TRNS, KC_TRNS, KC_TRNS  \
+  )
+};
+
+void matrix_scan_user(void) {
+  // jump to bootloaer when all keys are pressed
+  if (matrix_get_row(0) == 0b111 && matrix_get_row(1) == 0b111) {
+    reset_keyboard();
+  }
+};
```

Comment:
```suggestion
```
Don't need this. `VIA_ENABLE = yes` implies `BOOTMAGIC_ENABLE = lite`, so you can hold the key at (0,0) in the matrix (in this case, the top left or "A" key) while plugging in the board to reset.